android/app/src/main/kotlin/coop/polypoly/polypod/updatenotification/UpdateNotification.kt

```diff
@@ -5,23 +5,65 @@ import coop.polypoly.polypod.Preferences
 import coop.polypoly.polypod.R
 
 class UpdateNotification(private val context: Context) {
-    val id = context.resources.getInteger(R.integer.update_notification_id)
+    companion object {
+        data class MockData(var id: Int? = null)
+        val mockData = MockData()
```

Comment:
Bit unhappy with this, but after a lot of pondering, it's the best solution I could come up with for making this testable. Since notifications need to be loaded _before_ they are used, loading them in the constructor of an object (that does, however, not _hold_ its actual state) made most sense to me. But since we don't control initialisation outside the app, for the integration tests I needed _some_ mechanism to be able to control the ID.

Reply:
Why not make it a counter instead?

Reply:
(I suppose this was a misunderstanding of what `id` is for, that should be cleared up now - but I'll leave it unresolved just in case I misunderstood.)